snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/SnomedDescriptionAcceptabilityUpdateRequest.java

```diff
@@ -117,4 +167,14 @@ protected void doExecute(TransactionContext context, SnomedComponentDocument com
 		}
 	}
 	
+	private SnomedReferenceSetMember select(List<SnomedReferenceSetMember> members, final String modulePreference, final String secondaryModulePreference) {
+		return members.stream()
+			.filter(member -> member.getModuleId().equals(modulePreference))
+			.findFirst()
+			.orElse(members.stream()
+				.filter(member -> member.getModuleId().equals(secondaryModulePreference))
+				.findFirst()
+				.orElse(members.get(0)));
```

Comment:
From the three spots this method is called from, it is OK to use `members.get(0)` as the last fallback option as `members` is non-empty in all cases (first one is "if refset matches non-empty", second is the `else` branch of "if acceptability matches are empty", third is another `else` branch of "if active matches are empty").

To ensure that it stays that way, please add a precondition check to this method.